examples/visual_ssvep/01r__ssvep_viz.py

```diff
@@ -101,8 +101,9 @@
 axs[0].plot(freq1, psd1_mean[[0, 3], :].mean(0), color='b', label='30 Hz')
 axs[0].plot(freq2, psd2_mean[[0, 3], :].mean(0), color='r', label='20 Hz')
 
-axs[1].plot(freq1, psd1_mean[4, :], color='b', label='30 Hz')
-axs[1].plot(freq2, psd2_mean[4, :], color='r', label='20 Hz')
+# TODO fix: IndexError: index 4 is out of bounds for axis 0 with size 4
+# axs[1].plot(freq1, psd1_mean[4, :], color='b', label='30 Hz')
+# axs[1].plot(freq2, psd2_mean[4, :], color='r', label='20 Hz')
```

Comment:
Commenting out these lines of code is out of the scope of "Fixing the Docs" and messes with a visualization for SSVEP. This should be reverted and changed in another PR.

Reply:
Unfortunately the docs won't build without the change, I think I noticed the same issue a long time ago when trying to run the example locally.  Would you be able to help me figure out why it is giving the out of bounds error?

Reply:
The problem seems to be that this line

```python
>> psd1, freq1 = psd_welch(epochs['30 Hz'], n_fft=1028, n_per_seg=256 * 3, picks='all')
```

Is returning only 4 chans in its output object, whereas in this example there are 5 chans used (i.e. a muse 2016 with one extra auxiliary electrode over occipital cortex).

```python
>> psd1.shape
>> (105, 4, 129)
```

Trying to determine why this is happening...

Reply:
Ok got it: the channel type for the Aux electrode is not 'eeg'

```python
In [83]: raw.ch_names
Out[83]: ['TP9', 'AF7', 'AF8', 'TP10', 'POz', 'stim']
```

```python
In [82]: raw.get_channel_types()
Out[82]: ['eeg', 'eeg', 'eeg', 'eeg', 'misc', 'stim']
```

Easily fixed with 

```python
raw.set_channel_types({'POz': 'eeg'})
```

or something similar